ClosedXML/ClosedXML.csproj

```diff
@@ -67,11 +67,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
-    <PackageReference Include="Fody" Version="5.2.0">
+    <PackageReference Include="Fody" Version="6.3.0">
```

Comment:
Upgrading Fody to v6+ exposes us to a new license, which I do not support. See https://github.com/Fody/Home/blob/master/pages/licensing-patron-faq.md under the "What about open source projects consume or producing Fody weavers?" section. Is this upgrade completely necessary for Rider to work?

Reply:
Ah, yeah, I forgot about this... But the faq states it applies to versions 3.3.3 and higher, why do we have 5.2.0 here?

Reply:
That is a very good question! I was convinced that's when the license changed occurred!

Reply:
Yes, the announcement was made on or around 4 December 2018, at which point v3.3.2 was the latest version. Oops. Well, we can always maintain our own build based on 6.3.0, since it's MIT-licensed. But let's wait for the response on gitter.